airbyte-local-cli-nodejs/test/command.test.ts

```diff
@@ -0,0 +1,232 @@
+import {parseAndValidateInputs} from '../src/command';
+import {parseConfigFile} from '../src/utils';
+
+jest.mock('../src/utils');
+
+const defaultConfig = {
+  srcPull: true,
+  dstPull: true,
+  logLevel: 'info',
+};
+
+afterEach(() => {
+  jest.resetAllMocks();
+});
+
+describe('Check options conflict', () => {
+  it('should fail if using both --config-file and --src', async () => {
+    jest.spyOn(process, 'exit').mockImplementation(() => {
+      throw new Error('process.exit() was called by commander js');
+    });
+    const argv = ['./airbyte-local-cli', 'index.js', '--config-file', 'config-file-path', '--src', 'source-image'];
```

Comment:
Can the executable be set up to not require the `index.js` argument -- i.e. `./airbyte-local-cli --config-file ... --src ...`

Reply:
when the user runs the exec in terminal, they don't need `index.js`.

i need this is because i'm mocking the process argv which i think in node js that's how it works. tbh i'm still a bit loose on this nodejs knowledge so i don't know how this exactly works. https://github.com/tj/commander.js/blob/master/typings/index.d.ts#L727

```
/**
   * Parse `argv`, setting options and invoking commands when defined.
   *
   * Call with no parameters to parse `process.argv`. Detects Electron and special node options like `node --eval`. Easy mode!
   *
   * Or call with an array of strings to parse, and optionally where the user arguments start by specifying where the arguments are `from`:
   * - `'node'`: default, `argv[0]` is the application and `argv[1]` is the script being run, with user arguments after that
   * - `'electron'`: `argv[0]` is the application and `argv[1]` varies depending on whether the electron application is packaged
   * - `'user'`: just user arguments
   *
   * @example
   * ```
   * await program.parseAsync(); // parse process.argv and auto-detect electron and special node flags
   * await program.parseAsync(process.argv); // assume argv[0] is app and argv[1] is script
   * await program.parseAsync(my-args, { from: 'user' }); // just user supplied arguments, nothing special about argv[0]
   * ```
   *
   * @returns Promise
   */
```